### PR TITLE
WL-3968: Panel warning message if email or suggestions properties not set

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: src/webapp/WEB-INF/templates/home.handlebars [hunk 1]
```diff
@@ -21,21 +21,39 @@
     <div class="feedback-box" id="feedback-ask-help">
         <h3>{{translate 'ask_title'}}</h3>
         <ul>{{translate 'ask_explanation'}}</ul>
-        <p><a id="feedback-report-helpdesk-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
+        {{#if technicalToAddress}}
+            <p><a id="feedback-report-helpdesk-link" href="javascript:;">{{translate 'technical_link'}}</a></p>
+        {{else}}
+            <div class="feedback-error-message-panel-wrapper">
+                <div><img src="/library/image/silk/exclamation.png" /><span>{{translate 'no_technical_address_message'}}</span></div>
```
Comment (buckett) [inline src/webapp/WEB-INF/templates/home.handlebars]: Having an standalone `` tag is bad for accessibility reasons (it's not actually a picture someone wants to view), and instead you should use a background image on an existing div/span.

I think the tool already done this in other places.

**Comment by nicholaswilson100** [comment]
Made image background and squishisized.

**Comment by buckett** [comment]
Failing to merge.

**Comment by nicholaswilson100** [comment]
Sorry  in future I'll check to see if the request can still be merged

**Pull request closed without merging**